app/assets/stylesheets/_settings.scss

```diff
@@ -104,29 +104,37 @@ $block-grid-max: 8;
 // 4. Base Typography
```

Comment:
I think all of these different sections could be small files in a settings folder which would make it a bit easier to come back to and know where everything is